simplesat/request.py

```diff
@@ -45,9 +46,8 @@ class Request(object):
     {'allow_older': [], 'allow_any': ['MKL'], 'allow_newer': []}
     """
 
-    def __init__(self, modifiers=None):
-        self.jobs = []
-        self.modifiers = modifiers or ConstraintModifiers()
+    modifiers = attr(default=Factory(ConstraintModifiers))
```

Comment:
It's possible to add validators to these as well. Not required of course, but see the `ConstraintModifiers` class if you're interested in how that works.